.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -36,5 +37,5 @@ https://github.com/zfsonlinux/zfs/wiki/Buildbot-Options
 - [ ] I have updated the documentation accordingly.
 - [ ] I have read the [**contributing** document](https://github.com/zfsonlinux/zfs/blob/master/.github/CONTRIBUTING.md).
 - [ ] I have added [tests](https://github.com/zfsonlinux/zfs/tree/master/tests) to cover my changes.
-- [ ] All new and existing tests passed.
+- [ ] All new and existing tests passed _(Please think about using the draft PR feature if appropriate)_.
```

Comment:
After having filled out this form many times myself, my feeling is we're asking the wrong question here.  Why don't we change it and clearly ask what we really want to know when reviewing this.

```
[ ] I have locally run the ZFS Test Suite with this change applied.
```

Reply:
@behlendorf 
You know, this is what I was looking for... 

Maybe remove "locally", I think the emphasis on "I have run" is already a VERY big upgrade...

Reply:
I'm good with removing locally, that is a bit wordy!

Reply:
Yeah, funny enough it has the same problem as my initial version of things... :P

Reply:
Replaced with your suggestion without locally.